kolibri/content/api.py

```diff
@@ -0,0 +1,222 @@
+"""
+This module acts as the only interface point between other apps and the database backend for the content.
+It exposes several convenience functions for accessing content
+"""
+from kolibri.content import models as KolibriContent
+from uuid import UUID
+import os
+from bulk_update.helper import bulk_update
+from django.core.files import File as DjFile
+
+"""helper funcitons"""
```

Comment:
you mean put the helper function is_valid_uuid into kolibri/kolibri/utils or create a new utils module inside the content app?   Could you point me to one repeated patterns?